### DISCUSSION TITLE
Clearing audit records [May 20]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2185] [Discord Threads]
> 🙋‍♂️ **nick_neustroev**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Yes. clearing audit logs should be fine. note that, row comments, if added any - will also get cleared off.  
#  
> 🙋‍♂️ **nick_neustroev**  
#### Thanks! It helped, everything ok.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974520207706054678)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
